stories/ag/Filters/BathroomFilterCard.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import { action } from '@storybook/addon-actions'
+import { BathroomFilterCard } from 'react-ui-ag/src'
+
+const InlineBathroomFilterCardComponent = ({ theme }) => (
+  <BathroomFilterCard
+    className={theme.InlineCard}
+    applyButton={{ onClick: value => action('click')(`applied(${value})`) }}
+  />
+)
+
+InlineBathroomFilterCardComponent.propTypes = {
+  theme: PropTypes.object,
+}
+
+const ThemedInlineBathroomFilterCard = themed(['InlineCard'])(InlineBathroomFilterCardComponent)
+const InlineBathroomFilterCard = <ThemedInlineBathroomFilterCard />
+
+const DesktopBathroomFilterCardComponent = ({ theme }) => (
+  <BathroomFilterCard
+    className={theme.SearchFilter}
+    title=""
```

Comment:
Is there another way to show no text without leaving props with no value

Reply:
Yup. I changed out the defaults now. Good catch 👍